common/helpers/move/get-tag-lists.js

```diff
@@ -0,0 +1,21 @@
+const presenters = require('../../presenters')
+
+const getMoveUrl = require('./get-move-url')
+
+function getTagLists(move) {
+  const { profile } = move
+  const { assessment_answers: assessmentAnswers = [] } = profile || {}
+
+  const moveUrl = getMoveUrl(move.id)
+
+  const tagLists = {
+    tagList: presenters.assessmentToTagList(assessmentAnswers, moveUrl),
```

Comment:
It would be clearer if the properties and the tests were in the same order (e.g. importantTagList then tagList`)

Reply:
Reordered.